README.md

```diff
@@ -286,13 +289,13 @@ under a category in libcxx, or running a single test in `std` and `tr1`.
 These examples assume that your current directory is `C:\Dev\STL\out\x64`.
 
 * This command will run all of the test suites with verbose output.
-  + `ctest -V`
-* This command will also run all of the test suites.
+  + `ninja test extra-asan-tests`

```

Comment:
This still says "with verbose output", but is that true? `--verbose` is not being passed to `ninja` here, but we do in `azure-devops/run-tests.yml`. (I haven't checked what this does.)

Reply:
In that case, do we need `--verbose` in `azure-devops/run-tests.yml`?

Reply:
Ninja abbreviates the command for custom targets without verbose:
```
[0/1] C:\WINDOWS\system32\cmd.exe /C "cd /D C:\STL\out\x64...xx/test/std C:/STL/tests/std/tests C:/STL/tests/tr1/tests"-- Testing: 58732 tests, 32 workers --
SKIPPED: std :: tests/GH_003617_vectorized_meow_element:1 (1 of 58732)
SKIPPED: std :: tests/P2465R3_standard_library_modules:01 (2 of 58732)
```
which is fine for typical command-line use where you don't really care to see the full expansion, but for CI logs I prefer to see the full command:
```
C:\STL\out\x64>ninja -v test
[0/1] C:\WINDOWS\system32\cmd.exe /C "cd /D C:\STL\out\x64\tests && "C:\Program Files\Python313\python.exe" C:/STL/out/x64/tests/utils/stl-lit/stl-lit.py -o C:/STL/out/x64/tests/test_results.json -Dnotags=ASAN C:/STL/llvm-project/libcxx/test/std C:/STL/tests/std/tests C:/STL/tests/tr1/tests"
-- Testing: 58732 tests, 32 workers --
SKIPPED: std :: tests/GH_003617_vectorized_meow_element:1 (1 of 58732)
SKIPPED: std :: tests/P2465R3_standard_library_modules:01 (2 of 58732)
```

Reply:
That's an excellent reason, thanks. Abbreviated logs are the worst.